src/v2/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -218,7 +218,7 @@ const Conversation: React.FC<ConversationProps> = props => {
   return (
     <NoScrollFlex flexDirection="column" width="100%">
       <MessageContainer>
-        <Box>
+        <Box pb={[6, 6, 6, 0]}>
```

Comment:
This is the space needed to be padded to the end of the messages for the input field not to overlap it.